ansible/avena/roles/core/tasks/docker.yml

```diff
@@ -82,3 +82,55 @@
   become: yes
   command:
     cmd: python3 -m pip install -IU docker-compose
+
+- name: container-maintainer
```

Comment:
This fails during a deploy from scratch due to the fact that wireguard has not been setup yet so it cannot connect and download the images. Not merging until I get it figured out.

First solution I though of was to just enable the service but not start it and let it run the next day and assume everything has been setup by then, but doesn't feel the best